src/dart/NOTES.md

```diff
@@ -0,0 +1,26 @@
+<!-- markdownlint-disable MD041 MD033 -->
+
+<div align="center">
+
+[![Dart website](https://thum.io/get/width/800/crop/600/noanimate/https://dart.dev/)](https://dart.dev/)
+
+</div>
+
+> Dart is a client-optimized language for fast apps on any platform
+
+&mdash; [Dart programming language | Dart]
+
+This template will provide you with an environment ready to get started using
+Dart! You'll have access to the `dart` CLI. Check out [the Dart feature] that we
+use under the hood!
+
+## History
+
+This template was originally part of the [microsoft/vscode-dev-containers]
+project. It has since migrated here! ☄️

```

Comment:
Was there an agreement with the original authers?

Reply:
you know, i don't think there was... I'll `@`-ping the 3 most recent committers(?) and see what they think.

Reply:
https://github.com/devcontainers-community/templates/discussions/39